cdap-master/src/main/java/co/cask/cdap/data/runtime/main/transaction/TransactionHttpService.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data.runtime.main.transaction;
```

Comment:
Oh I see. If we introduce an HTTP handler in the tx service, then those endpoint should probably move here. Can you add a TODO and open a Jira for that? (no need to move them in this PR)